lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_hotifx_prechecks.rb

```diff
@@ -50,13 +50,13 @@ def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :version_name,
                                        env_name: 'FL_ANDROID_HOTFIX_PRECHECKS_VERSION',
-                                       description: 'The version to work on', # a short description of this parameter
+                                       description: 'The hotfix version number to create',
```

Comment:
It was always unclear to me when using this lane if this was supposed to be the version we want to cut a hotfix from (eg 17.7), letting the action goes the hotfix version number to use … or the hotfix version number we want (eg 17.7.1). Wanted to make clear in the doc that it's the latter.

Reply:
This is super nice to have!!